### ISSUE TITLE
Support for jqwik

### ISSUE THREAD

**Issue by ragunathjawahar** [approvals/ApprovalTests.Java#133]
I am trying to use ApprovalTests with [jqwik](https://jqwik.net) property based testing library. However, I run into this exception.

```
Could not find Junit/TestNg TestCase you are running, supported frameworks: Junit3, Junit4, Junit5, TestNg
java.lang.RuntimeException: Could not find Junit/TestNg TestCase you are running, supported frameworks: Junit3, Junit4, Junit5, TestNg
	at org.approvaltests.namer.AttributeStackSelector.selectElement(AttributeStackSelector.java:62)
	at com.spun.util.tests.TestUtils.getCurrentFileForMethod(TestUtils.java:175)
	at com.spun.util.tests.TestUtils.getCurrentFileForMethod(TestUtils.java:168)
	at org.approvaltests.namer.StackTraceNamer.<init>(StackTraceNamer.java:15)
	at org.approvaltests.Approvals$1.load(Approvals.java:45)
	at org.approvaltests.Approvals$1.load(Approvals.java:42)
	at org.approvaltests.Approvals.createApprovalNamer(Approvals.java:247)
	at org.approvaltests.core.Options$FileOptions.getNamer(Options.java:90)
	at org.approvaltests.Approvals.verify(Approvals.java:178)
	at org.approvaltests.Approvals.verify(Approvals.java:54)
	at org.approvaltests.Approvals.verify(Approvals.java:50)
	at com.gildedrose.GildedRoseJqwikApprovalTest.goldenMaster(GildedRoseJqwikApprovalTest.kt:35)
```

I find property tests to be quite valuable while attempting to bring legacy code bases under tests. Would like to hear your thoughts on this.

If we come to an agreement, I can raise a PR for this feature request. LMK. Many thanks in advance.

**Example**
```kotlin
@Property(seed = "6348610496668862340", tries = 1)
fun goldenMaster(
  @ForAll @From("name") names: List<String>,
  @ForAll @From("sellIn") sellIns: List<Int>,
  @ForAll @From("quality") qualities: List<Int>,
) {
  // given
  val items = names.zip(sellIns).zip(qualities) { (name, sellIn), quality -> Item(name, sellIn, quality) }
  val gildedRose = GildedRose(items.toTypedArray())

  // when
  for (i in 0..50) {
    gildedRose.updateQuality()
  }

  // then
  val gildedRosesSnapshot = gildedRose
    .items
    .joinToString("\n") { it.toString() }
  Approvals.verify(gildedRosesSnapshot)
}
```

**Comment by LarsEckart** [comment]
We are interested in exploring this. Are you up for a quick video call? What's your timezone?

**Comment by ragunathjawahar** [comment]
I am interested, my timezone is UTC +5:30. You can reach out to me at ragunathjawahar@gmail.com. We can find a time that works best for both of us this week or next.

**Comment by ragunathjawahar** [comment]
@LarsEckart any update on this?

**Comment by LarsEckart** [comment]
hey,
sorry, I completely forgot about it. Our next session is probably on Monday 29th, once I know the time, I'll also send out an invite to you, hope the date&time will work for you too :)

**Comment by ragunathjawahar** [comment]
Sounds like a plan. Looking forward to it!

**Comment by LarsEckart** [comment]
resolved. we talked about using CombinationApprovals and fixed issues with kotlin test names (with spaces) and multi-module gradle projects. released 10.3.0
